recipes/goss/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "goss" %}
```

Comment:
Why use a different name on PYPI and conda?

Reply:
Consistency is important because requirements.txt so that requirements.txt files can be used to create a conda environment without updating the package names.

Reply:
OK, I have now changed the same so that it matches the one on pypi, see https://github.com/conda-forge/staged-recipes/pull/22029/commits/40707725c680aba8c9ee75f7f211d85520a47161